Dockerfile

```diff
@@ -3,8 +3,6 @@ MAINTAINER Troy de Freitas "troy.defreitas@uptake.com", Nick Paras "nick.paras@u
 
 RUN apt-get update -y
 
-RUN pip install flask boto3 gunicorn python-debian
```

Comment:
we'd still need gunicorn right? doesn't look like that one got moved into the `setup.py`

Reply:
Yeah, good catch. We don't want Gunicorn in `setup.py` but definitely in the Dockerfile for sure.